docs/smartagent-deprecation-notice.md

```diff
@@ -0,0 +1,7 @@
+# Deprecation Notice for SignalFx Smart Agent
+
+Smart Agent is deprecated in conjunction with the release of Splunk Observability Cloud:
+
+* Smart Agent continues to function and is still supported by Splunk.
+* Apart from the [documented exceptions](https://docs.splunk.com/Observability/get-started/migrate/migrate-to-otel.html) where integration with Splunk Observability Cloud still requires deploying Smart Agent, Smart Agent functionality is replaced by the [Splunk Distribution of OpenTelemetry Connector](https://docs.splunk.com/Observability/get-started/get-data-in/get-data-in.html).
+* Splunk is committed to further development of OpenTelemetry, and might remove Smart Agent in a future version.
```

Comment:
I'm not sure what "might remove Smart Agent in a future version" means. Suggest "may discontinue support for Smart Agent in the future and will provide customers advance notice of such discontinuation."

Reply:
Third bullet now says this:
"Splunk is committed to further development of OpenTelemetry, and may discontinue support for Smart Agent in the future after providing customers advance notice of such discontinuation."